drafts/ecc-zf-split-plus-voting.rst

```diff
@@ -0,0 +1,103 @@
+
+Specification
+========
+
+Voting System Requirements
+-----------------
+
+Here are the general properties of the mandated voting mechanism. I don’t want to specify the technical implementation details, since I believe this is a job suited for the engineers building this system.
+
+1. Voting should be private.
+2. Only Zec holders can vote.
+3. Voting should happen on-chain.
+4. In order to vote, you must lock your zec so it cannot be spent for a period of time. This is to force the voters to have ‘skin in the game’ and prevent someone nefarious from buying a lot of zec just before an election and then dumping it immediately after.
+5. Voters can choose how long to lock their zec, and their voting power is proportional to the time that the zec is locked. For example, someone who votes with 10 zec and locks it for 6 months would have the same voting power as someone who votes with 20 zec and locks it for 3 months. Of course there must be a maximum lock time, perhaps a year, to prevent anyone from getting ‘infinite’ voting power by locking their zec permanently.
+6. The final results of the vote should be transparent to and verifiable by everyone.
+7. The system should be totally open and allow anyone/any organization to compete for funding to develop zcash.
+
+Transparency and Accountability for the ECC and the Zcash Foundation
+-----------------
+
+These requirements would apply to both the company and the foundation. The mandate is to adhere to these accountability requirements originally put forward by the foundation:
+
+* Monthly public developer calls, detailing current technical roadmap and updates
+* Quarterly tech roadmap reports and updates
+* Quarterly financial reports, detailing spending levels/burn rate and cash/ZEC on hand
+* A yearly, audited financial report akin to the Form 990 for US-based nonprofits
+* Yearly reviews of organization performance, along the lines of our State of the Zcash Foundation report [1]
+
+Requirements Specifically for the ECC
+-----------------
+
+Motivated by the Foundation’s proposal that the ECC become a non-profit ( https://www.zfnd.org/blog/dev-fund-guidance-and-timeline/), I am going to list general requirements for the ECC to abide by if they choose to receive funds and work on behalf of zec holders.
+
+1. Because I share the foundation’s concern that the ECC could be “beholden to its shareholders”, I am mandating that the ECC should be working in the service of the Zcash community and **shall serve no other masters**. The original investors/founders who are not still working in the service of the Zcash community should not have control over the use of the new dev funds.
+2. The revenue received from the Dev Fund should not be used to give further rewards to, even indirectly, the investors, founders, or shareholders of the ECC, who are not working on Zcash after the first halving. **They have already received the founders reward and this new dev fund is not supposed to further benefit them.**
```

Comment:
Under ECC's current structure, I think the first requirement is impossible to meet as phrased: as a partnership, ECC is inherently "beholden to its shareholders", and its board is determined by shareholders. Here's an idea for alternative phrasing (jointly for these two requirements):

*ECC must undertake a firm obligation to use the Dev Fund only in support of the Zcash cryptpcurrency and its community, and to not distribute the Dev Fund proceeds to its partners ("shareholders"), other than*
*1. In fair compensation for ongoing work*
*2. For covering pass-through tax obligations to partners caused by ECC's receipt of the Dev Fund*
*This obligation must be made irrevocable within ECC's corporate governance (i.e., its Operating Agreement and Board of Directors), and backed by a contractual obligation to the Zcash Foundation.*